pal/system-info/enum-type/feature.ts

```diff
@@ -0,0 +1,12 @@
+export enum Feature {
+    CANVAS = 'CANVAS',
+    GL = 'GL',
```

Comment:
Use WEBGL instead please. Also please add GLES2, GLES3, VULKAN and METAL for native platforms.